services/src/main/resources/org/keycloak/protocol/oidc/endpoints/3p-cookies-step1.html

```diff
@@ -35,7 +40,7 @@
 
       function signalSupport(isSupported) {
         // Signal 3rd-party access support to the page embedding this iframe.
-        window.parent.postMessage(isSupported ? "supported" : "unsupported", "*");
+        window.parent.postMessage( isSupported ? "supported" : "unsupported", "*");
```

Comment:
Can we get rid of these white-space changes?
```suggestion
        window.parent.postMessage(isSupported ? "supported" : "unsupported", "*");
```